terraform/azure-pipelines.yml

```diff
@@ -26,8 +26,14 @@ stages:
             value: $[variables['Build.SourceBranchName']]
           - name: TARGET
             value: $[variables['System.PullRequest.TargetBranch']]
+          - name: IS_TAG
+            value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/')]
         steps:
-          - bash: python terraform/pipeline/workspace.py
+          - bash: |
+              python terraform/pipeline/workspace.py
+
+              TAG_TYPE=$(python terraform/pipeline/tag.py)
+              echo "##vso[task.setvariable variable=tag_type;isOutput=true]$TAG_TYPE"
```

Comment:
These lines could be normalized to move the `TAG_TYPE` and `echo` in `tag.py` similar to how it works in `workspace.py`, but not a priority for this PR.

Reply:
I agree. Great, I'll create an issue to come back to this later.